build-conventions/src/main/kotlin/com/leinardi/forlago/ext/CommonExtensionExt.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2024 Roberto Leinardi.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.leinardi.forlago.ext
+
+import com.android.build.api.dsl.CommonExtension
+import com.android.build.api.dsl.ManagedVirtualDevice
+import com.leinardi.forlago.model.DeviceConfig
+import org.gradle.kotlin.dsl.create
+import org.gradle.kotlin.dsl.invoke
+
+/**
+ * Configure project for Gradle managed devices
+ */
+@Suppress("UnstableApiUsage")
+fun CommonExtension<*, *, *, *, *, *>.configureGradleManagedDevices(
+    minApiLevel: Int,
+    maxApiLevel: Int,
+    device: String = "Pixel 2",
+    systemImageSource: String = "google",
+) {
+    val allDevices: List<DeviceConfig> = (minApiLevel.coerceAtLeast(MIN_SUPPORTED_GMD_API_LEVEL)..maxApiLevel).map { apiLevel ->
```

Comment:
👍 if now we don't create GMDs with API version lower than 27 I would say we can remove this from `gradle.properties`:
```
android.experimental.testOptions.managedDevices.allowOldApiLevelDevices=true
```

Reply:
It is still needed until our minSDK is < 27 or it will fail when running certain tasks (I encountered the issue already)